### ISSUE TITLE
Change the default splash screen 

### ISSUE THREAD

**Issue by dishant-yadav** [dishant-yadav/ReMed#1]

**Comment by TheCoderAdi** [comment]
Hello, hope you are doing great ,
What type of image you want in the splash screen.

**Comment by TheCoderAdi** [comment]
Hello @dishant-yadav , I have to add this image ?
![s](https://user-images.githubusercontent.com/111285422/231443275-ff1a0121-1a43-4bab-ba0f-9a44abdee890.png)

**Comment by dishant-yadav** [comment]
Is this the same one which i send it earlier?

**Comment by TheCoderAdi** [comment]
No , It is the new one

**Comment by dishant-yadav** [comment]
Yeah, you can add this. Just make sure that the Hexagon is in the middle of the screen

**Comment by TheCoderAdi** [comment]
Okay
